cli/command/inspect/inspector_test.go

```diff
@@ -219,3 +226,19 @@ func TestIndentedInspectorRawElements(t *testing.T) {
 		t.Fatalf("Expected `%s`, got `%s`", expected, b.String())
 	}
 }
+
+// moby issue #32235
```

Comment:
Minor thing, but It would be good to be more explicit here by using the short-form url moby/moby#32235.  "moby" is a github org and project, so issue #32235 isn't specific enough. It might make sense today, but I'm not sure it will be so clear in a few months.